ola/AutoStart.cpp

```diff
@@ -110,7 +110,11 @@ TCPSocket *ConnectToServer(unsigned short port) {
     // Try to start the server, we pass --daemon (fork into background) and
     // --syslog (log to syslog).
     execlp("olad", "olad", "--daemon", "--syslog",
-           reinterpret_cast<char*>(NULL));
+    #ifdef __FreeBSD__
+                    reinterpret_cast<char*>(0));
```

Comment:
Can we align this, and the line below with the ( above please.

Reply:
Done, with some more occurences :)